vignettes/ar-grid.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Create A&R Grid"
+output:
+  rmarkdown::html_vignette
+resource_files:
+   - package_manual.pdf
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Create A&R Grid}
+---
+
+```{r, echo = FALSE, message = FALSE, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  out.width = "100%"
+)
+```
+
+```{r, message = FALSE}
+library(metalite)
+library(r2rtf)
+library(dplyr)
+```
+
+## Overview
+
+In this document, we illustrate how to create the A&R grid by `metalite`.
+
+## Build a Meta Data
+
+### Step 1: input the population and observation datasets.
+```{r}
+my_meta <- meta_adam(population = r2rtf_adsl,
+                     observation = r2rtf_adae)
+```
+
+
+### Step 2: create statistical analysis plans.
+
+```{r}
+my_plan <- plan(analysis = "ae_summary", population = "apat", 
+             observation = c("wk12", "wk24"), parameter = "any;rel;ser") |>
+  
+  add_plan(analysis = "ae_specific", population = "apat", 
+           observation = c("wk12", "wk24"), 
+           parameter = c("any", "aeosi", "rel", "ser"))
+```
+
+### Step 3: feed in the analysis plan to the existing meta data.
```

Comment:
Is it `meta data` or `metadata`? Need to spell consistently throughout the package.

Reply:
Metadata is used in this package. So I use it throughout the package.